From b28c882889cbea9be3748d3dee1b21ba6eb90b66 Mon Sep 17 00:00:00 2001
From: Arthur Williams <taaparthur@gmail.com>
Date: Fri, 25 Sep 2020 04:52:57 +0000
Subject: [PATCH xserver 04/16] include: Increase the number of max. input
 devices to 256.

Extending the decade old f0124ed93, to increase the number of input
devices from 40 to 256. 40 translates at most 9 MD, while 256 will allow
63 MD. It is an arbitrary number, but people are hitting the current
limit under reasonable conditions.

Bugzilla: https://bugs.freedesktop.org/show_bug.cgi?id=64793
Signed-off-by: Arthur Williams <taaparthur@gmail.com>

(cherry picked from commit fe439596b99db65bbae35eff1ea0b79db167f8d6)
---
 include/misc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/misc.h b/include/misc.h
index 14920c3c3..9b422e53b 100644
--- a/include/misc.h
+++ b/include/misc.h
@@ -91,7 +91,7 @@ OF THIS SOFTWARE.
 #define LIMITCLIENTS	256     /* Must be a power of 2 and <= MAXCLIENTS */
 #define MAXEXTENSIONS   128
 #define MAXFORMATS	8
-#define MAXDEVICES	40      /* input devices */
+#define MAXDEVICES	256      /* input devices */
 #define GPU_SCREEN_OFFSET 256
 
 /* 128 event opcodes for core + extension events, excluding GE */
-- 
2.28.0

